README.md

```diff
@@ -1,24 +1,33 @@
+### Note: This collection is not officially supported by Red Hat.
 
-Ansible Roles for interacting with the Red Hat Satellite API, combining different modules into workflows. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) (FAM).
 
-# What this project is not?
-- This is not an Ansible Collection. The roles present here are standalone roles.
-- This will not be published in Ansible Galaxy.
-- This project will not accept Ansible modules. If you are writing a new module, consider contributing it to [FAM](https://github.com/theforeman/foreman-ansible-modules).
-- The roles here will not be shipped with [FAM](https://github.com/theforeman/foreman-ansible-modules). If you like your role to be shipped with foreman-ansible-modules, consider contributing it to FAM using the rules listed in [to be updated](https://github.com/theforeman/foreman-ansible-modules/issues/646)
+Ansible Roles, Playbooks (and Tower Configuration metadata) for installing, configuring and interacting with the Red Hat Satellite. The roles will be created using [foreman-ansible-modules] where possible(https://github.com/theforeman/foreman-ansible-modules) (FAM).
 
-## Rules for contributing Roles to this project
+# Contribution Guidelines
 
-* The use cases supported by the roles should be beneficial for many users.
-* Input variables externalized in `defaults` folder of roles for easy reuse.
-* The roles should be compatible with the currently (upstream) supported Ansible versions, 2.7+ at the time of writing.
-* The roles should be compatible with any Python supported by Ansible upstream (2.7, 3.5, 3.6, 3.7 at the time of writing), however, roles usually don’t require any special Python features (in contrast to modules).
-* The roles should be compatible with the latest released version of the [FAM](https://github.com/theforeman/foreman-ansible-modules) collection (0.3.0 at the time of writing). While defining a dependency on an older FAM release is possible, it is discouraged.
-* The roles shall not use hammer via command/shell or the uri module to talk to the API directly.
+- This collection is a collaborative development environment.
+- This collection may not accept all Ansible modules proposed. If you are writing a new module related to Foreman, consider contributing it to [FAM](https://github.com/theforeman/foreman-ansible-modules). For modules that have other source projects, we may request you to open a pull request against respective projects.
+
+
+## Roles
+
+* The roles written in this collection should be extensible, reusable and idempotent.
+* The input variables should be externalized in the `defaults` directory of the roles allowing for easy reuse.
+* The variables that are required by the role but not required by user to be modified, should reside in `vars` directory.
+* The roles should be compatible with the current (upstream) supported Ansible versions, 2.9+(?) at the time of writing.
```

Comment:
Today it's 2.7+ upstream (see https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html), but 2.7 does not support collections and we've been not officially supporting it in FAM for a while now because of that.

I'd say 2.8+ is a good target, but have no problem if we want to say 2.9 is minimum either.

Reply:
Understood. I think we will stick to 2.9 then. I recall others on the call were ok with that being minimum as well. Thanks.